package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This is a metadata file you see with node/javascript projects.
It contains important information about the project like dependencies, authors, and scripts.
This meta is used by command line tools like `npm` to help you run the build and run the project.

It's somewhat similar to Makefiles in C++ or something like that.

Reply:
Okay cool, I'm familiar with the basics behind package managers. I haven't used them much yet, but I understand WHY they are used. Do we need to select a package manager to use? Are we using `npm` for this project?

Reply:
Yeah, I used `npm` for this because it's the most vanilla option. But I don't mind using something more modern like `pnpm` or whatever if you feel like it.